examples/README.md

```diff
@@ -0,0 +1,13 @@
+# BluePyEModel Examples
+
+This directory includes various examples demonstrating how to use BluePyEModel:
+
+- [**L5PC**](./L5PC/README.rst): Provides a guide for setting up and running the E-Model building pipeline locally on a personal computer or on a Slurm cluster. This is an example based on the juvenile (P14) rat somatosensory cortex layer 5 thick-tufted pyramidal cell (L5TPC) e-model optimisation of the [SSCxEModelExamples](https://github.com/BlueBrain/SSCxEModelExamples/tree/main/optimization) repository.
+
+- [**nexus**](./nexus/README.md): Demonstrates the use of BluePyEModel through the Nexus access point.
+
+- [**others**](./others/README.rst): Contains various examples demonstrating different functionalities and use cases.
+    - [**ICSelector**](./others/icselector/icselector_example.py): Selects ion channel models using gene mappings.
+    - [**local2nexus**](./others/local2nexus/README.md): Exports a locally built e-model to Nexus.
+    - [**memodel**](./others/memodel/README.md): Retrieves, modifies, analyzes, and uploads an MEModel.
```

Comment:
Could you reorder and update the text to something like:
- [**run_emodel**](./others/run_emodel/README.rst): Runs an EModel simulation on BlueCelluLab to explore the single cell behaviour.
- [**memodel**](./others/memodel/README.md): Creates, runs, analyses, and uploads a MEModel.
-  [**local2nexus**](./others/local2nexus/README.md): Stores a locally built EModel (using LocalAccessPoint) to the [BlueBrain Nexus](https://github.com/BlueBrain/nexus) knowledge graph.
- [**ICSelector**](./others/icselector/icselector_example.py): Selects ion channel models using gene mappings; assigns optimisation parameter bounds and fixed values for a model. 


You can edit the text in the [README](https://github.com/BlueBrain/BluePyEModel/pull/164/files#diff-dc01789daf4e64b0597e384f075070f9e34f5310faeb82363c36554b9257dca0) inside the folder if necessary.

Reply:
done in the last commit